.gitlab-ci.yml

```diff
@@ -37,3 +38,13 @@ end:
   only:
     refs:
       - external_pull_requests
+
+notify_failure:
+  stage: notify
+  script:
```

Comment:
The script must never fail.

Reply:
It doesn't matter if we add allow_failure. The script is always in the last stage of the pipeline. It only runs if one of the previous stages has failed. There is no scenario where the pipeline is successful, but the notify stage fails. Either the pipeline fails and the notify stage runs and is successful, or the pipeline fails and the notify stage fails as well.

I'll add the flag in case this scenario changes in the future.